gm4_pig_tractors/data/gm4_pig_tractors/advancements/guidebook/page_0.json

```diff
@@ -0,0 +1,136 @@
+{
+  "display": {
+    "icon": {
+      "item": "minecraft:iron_hoe",
+      "nbt": "{CustomModelData:3420001}"
+    },
+    "title": {
+      "text": "Check your guidebook!",
+      "color": "#4AA0C7",
+      "italic": true
+    },
+    "description": "Pig Tractors",
+    "frame": "goal",
+    "show_toast": true,
+    "announce_to_chat": false,
+    "hidden": true
+  },
+  "parent": "gm4_guidebook:root",
+  "criteria": {
+    "requirement": {
+      "trigger": "minecraft:tick",
+      "conditions": {
+        "player": [
+          {
+            "condition": "minecraft:inverted",
+            "term": {
+              "condition": "minecraft:entity_properties",
+              "entity": "this",
+              "predicate": {
+                "player": {
+                  "gamemode": "spectator"
+                }
+              }
+            }
+          },
+          {
+            "condition": "minecraft:value_check",
+            "value": {
+              "type": "minecraft:score",
+              "target": {
+                "type": "minecraft:fixed",
+                "name": "gm4_guidebook"
+              },
+              "score": "load.status"
+            },
+            "range": 1
+          },
+          {
+            "condition": "minecraft:value_check",
+            "value": {
+              "type": "minecraft:score",
+              "target": {
+                "type": "minecraft:fixed",
+                "name": "gm4_pig_tractors"
+              },
+              "score": "load.status"
+            },
+            "range": 1
+          },
+          {
+            "condition": "minecraft:entity_properties",
+            "entity": "this",
+            "predicate": {
+              "vehicle": {
+                "type": "minecraft:pig"
```

Comment:
This also triggers when sitting on a chair, so I think this should either ban the pigs with tag `gm4_chairs` specifically, or just ignore pigs with `NoAI`